Sources/NIOCore/ByteBuffer-int.swift

```diff
@@ -113,6 +113,25 @@ extension ByteBuffer {
             self.setBytes(ptr, at: index)
         }
     }
+
+    /// Returns the integer at the current reader index without advancing it.
+    ///
+    /// This method is equivalent to calling `getInteger(at: readerIndex, ...)`
+    ///
+    /// - Parameters:
+    ///   - endianness: The endianness of the integer (defaults to big endian).
+    ///   - as: The desired `FixedWidthInteger` type (optional parameter).
+    /// - Returns: An integer value deserialized from this `ByteBuffer` or `nil` if the bytes are not readable.
+    @inlinable
+    public func peekInteger<T: FixedWidthInteger>(
+        endianness: Endianness = .big,
+        as: T.Type = T.self
+    ) -> T? {
+        guard let result = self.getInteger(at: self.readerIndex, endianness: endianness, as: T.self) else {
+            return nil
+        }
+        return result
```

Comment:
We don't need the `guard` here. The formatter was complaining because of the explicit return where it wasn't necessary:

```suggestion
        self.getInteger(at: self.readerIndex, endianness: endianness, as: T.self)
```